PowerEditor/src/Notepad_plus_Window.cpp

```diff
@@ -18,7 +18,7 @@
 #include <shlwapi.h>
 #include "Notepad_plus_Window.h"
 
-const TCHAR Notepad_plus_Window::_className[32] = TEXT("Notepad++");
+const wchar_t Notepad_plus_Window::_className[32] = L"Notepad++";

```

Comment:
@donho 
Now (C++11+) you can remove this line from the .cpp and initialize directly in the Notepad_plus_Window.h:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/4d8ef955e28e2164224946261e5dcdb00f313f87/PowerEditor/src/Notepad_plus_Window.h#L112
`static constexpr wchar_t _className[] = TEXT("Notepad++");`